README.md

```diff
@@ -93,6 +93,11 @@ Get url for the administration:
 open http://$(docker-compose port nginx 80)/admin
 ```
 
+Before logging in to the box configuration administrative interface. First create an administrative user.
```

Comment:
Before logging into the box configuration administrative interface, create an administrative user:

Reply:
Fixed